client/scss/components/_numbered_list.scss

```diff
@@ -53,6 +53,7 @@ $numbered-list-number-size-large: 96px;
 }
 
 .numbered-list__number {
+  font-family: $wellcome-bold, $wellcome-bold-subset, $wellcome-bold-base;
```

Comment:
After a conversation with @Norvard, I wondered whether what we should change what is currently 'WB1' to this size. The current 'WB1' is not in use anywhere on the site yet – it's so big (156px) that it is only ever going to be used in extreme/unusual display settings and possibly/probably doesn't belong to the overall type system (or it could be included as 'WB0' if/when it was needed).